### ISSUE TITLE
[Feature] Add Shortcuts to switch between 3 modes

### ISSUE THREAD

**Issue by christianhans** [sn-extensions/simple-markdown-editor#22]
I suggest adding simple shortcuts to switch between Edit, Split and Preview.

**Mac:**
CMD + 1: Edit
CMD + 2: Split
CMD + 3: Preview

**Linux/Windows:**
CTRL + 1: Edit
CTRL + 2: Split
CTRL + 3: Preview

**Comment by mobitar** [comment]
What makes shortcuts really hard to implement is interfering with system shortcuts. Even knowing which system uses what shortcut is hard. With browsers (Chrome) on Mac, Cmd + 1/2/3 switches between the first three tabs open. So that wouldn't work.

**Comment by christianhans** [comment]
Agreed. I think in the long all shortcuts (global and extension-specific) need to be configurable. I envision something along the lines of how Visual Studio Code has a configuration file for this.
